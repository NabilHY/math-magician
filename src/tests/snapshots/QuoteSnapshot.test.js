import renderer from 'react-test-renderer';
import Quote from '../../component/quote';

describe('Quote component renders correctely', () => {
  it('Quote page matches snapshot', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
