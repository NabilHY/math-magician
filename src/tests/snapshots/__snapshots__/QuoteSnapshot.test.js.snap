// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component renders correctely Quote page matches snapshot 1`] = `
<h2>
  Mathematics is not about numbers, equations, computaions, or algorithms: it is about understanding. William Paul Thurston
</h2>
`;
